Documentation/event-reference.md

```diff
@@ -89,11 +89,24 @@ If the required `xdm` key is not present in the event data payload, the event is
 | --- | ---------- | -------- | ----------- |
 | xdm | <code>[String:&nbsp;Any]</code> | Yes | XDM formatted data; use an `XDMSchema` implementation for better XDM data ingestion and data format control. |
 | data | <code>[String:&nbsp;Any]</code> | No | Optional free-form data associated with this event. |
-| datastreamIdOverride | `String` | No | Optional Datastream identifier used to override the default datastream identifier set in the Edge configuration. |
-| datastreamConfigOverride | <code>[String:&nbsp;Any]</code> | No | Optional Datastream configuration used to override individual settings from the default datastream configuration. |
+| config | <code>[String:&nbsp;Any]</code> | No | Optional config overrides. |
```

Comment:
nit: 
- Optional config settings. Find the available keys for `config` below.
- Optional request parameters. Find the available keys for `request` below.